embulk-input-s3/src/main/java/org/embulk/input/s3/HttpProxy.java

```diff
@@ -14,15 +14,15 @@
 {
     private final String host;
     private final Optional<Integer> port;
-    private final boolean https;
+    private final Optional<Boolean> https;
```

Comment:
I used `Task` interface to implement this `HttpProxy`. It seems that `@JsonProperty.defaultValue` can be used by jackson v2.6 or later. Since Embulk now uses v2.5, I could not use it. 9eec90b02beea18c6daf29167d6b03ba4b23e592